src/main/java/org/openrewrite/kotlin/tree/K.java

```diff
@@ -577,8 +576,28 @@ final class Constructor implements K, Statement, TypedTree {
         Markers markers;
         J.MethodDeclaration methodDeclaration;
         Space colon;
```

Comment:
We should check if we can annotate both `colon` and `constructorInvocation` as `@Nullable` and if possible also as `@JsonIgnore` so that we don't continue serializing them, as we otherwise never will be able to derelease these fields.

Reply:
The fields should of course also be annotated as `@Deprecated`.

Reply:
We don't need to preserve the fields at all. Just need to have a constructor that reallocates their data to the `JLeftPadded`.

Reply:
As discussed yesterday we should verify that this also works as expected in a SaaS-like environment (with the corresponding Jackson config).